docs/src/pages/components/data-grid/components/CustomPaginationGrid.js

```diff
@@ -19,7 +19,7 @@ function CustomPagination() {
       className={classes.root}
       color="primary"
       count={state.pagination.pageCount}
-      page={state.pagination.page + 1}
+      page={state.pagination.currentPage + 1}
```

Comment:
why not just page? We know already it's the current one 😄

Reply:
It was to avoid `state.page.page`
But now that we have a single state for both hooks I can rename it :+1:

Reply:
Would ne nice to keep `page`. The diff will be smaller.